core/src/apps/cardano/helpers/utils.py

```diff
@@ -42,3 +42,8 @@ def format_optional_int(number: Optional[int]) -> str:
         return "n/a"
 
     return str(number)
+
+
+def append_unique(lst: List[Any], item: Any) -> None:
```

Comment:
this should be `lst: List[T], item: T` for some `T = TypeVar(...)`

Reply:
https://github.com/trezor/trezor-firmware/pull/1493/commits/3d6644da227656edc3a5a29cf970e05c9f9d4c25